config/database.ts

```diff
@@ -0,0 +1,54 @@
+/**
+ * Config source: https://git.io/JesV9
+ *
+ * Feel free to let us know via PR, if you find something broken in this config
+ * file.
+ */
+
+import Env from '@ioc:Adonis/Core/Env';
```

Comment:
Sure give you everetyhing huh :P Instead of using dotenv

Reply:
In a fresh adonisjs installation, the root directory of the application contains a **.env** file that defines common environment variables and this file can not be tracked by version control, so the **import Env from '@ioc:Adonis/Core/Env'** calls the .env that is in the root directory with environment variables. 
Now when working in a team, they can not see the **.env** file instead they see the **.env.example** file which contains placeholder values for environment variables. so all they do is type this command **cp .env.example .env** on their terminals and configure .env locally on their computers.

Reply:
Yeah I get you :D npm dotenv package is the same. Maybe what they use behind this package.
We use a combination of .env, and injecting during runtime